OvmfPkg/OvmfPkgX64.dsc

```diff
@@ -776,6 +776,18 @@
       NULL|OvmfPkg/Library/MpInitLibDepLib/PeiMpInitLibUpDepLib.inf
   }
 
+  #
+  # Cc Measurement PPI for Td guest
+  #
+!if $(CC_MEASUREMENT_ENABLE) == TRUE
+  OvmfPkg/Tcg/TdTcg2Pei/TdTcg2Pei.inf {
+    <LibraryClasses>
+      NULL|OvmfPkg/IntelTdx/TdxHelperLib/PeiTdxHelperLib.inf

```

Comment:
BTW: Is there a specific reason why TdxHelperLib hasn't a library class?

Reply:
No specific reason, updated now.

Reply:
Nice cleanup, thanks.